itest/mint_batch_stress_test.go

```diff
@@ -183,10 +183,7 @@ func mintBatchStressTest(
 	require.NoError(t, err)
 	require.Len(t, uniRoots.UniverseRoots, groupCount)
 
-	err = AssertUniverseRoot(
-		t, alice, groupBalance, nil, collectGroupKey,
-	)
-	require.NoError(t, err)
+	AssertUniverseRoot(t, alice, groupBalance, nil, collectGroupKey)
```

Comment:
Wouldn't this just fail the test either way?

Reply:
Yeah, but before it was a mixture of `require` and returning an error. And now we just do `t.Fatalf()` directly instead of returning the error and then doing `require.NoError()` outside.